packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -39,13 +41,51 @@ export default class {
     graphApi: GraphApi,
     params: {
       accountId: string;
-      schema: DataType;
+      // we have to manually specify this type because of 'intended' limitations of `Omit` with extended Record types:
+      //  https://github.com/microsoft/TypeScript/issues/50638
+      //  this is needed for as long as DataType extends Record
+      schema: Pick<DataType, "kind" | "title" | "description" | "type"> &
+        Record<string, any>;
     },
   ): Promise<DataTypeModel> {
-    const { data: identifier } = await graphApi.createDataType(params);
+    /** @todo - get rid of this hack for the root account */
```

Comment:
The workspace account doesn't have an associated user, so I had to grab it from the env-var because `getUserByAccountId` fails. Obviously this is less than ideal.

Reply:
Not sure what we want to do for this PR, I don't want to leave the TODO as is